src/routes/changelog/(entries)/2025-01-31.markdoc

```diff
@@ -1,22 +1,21 @@
 ---
 layout: changelog
-title: New database metrics and upload safeguards
+title: Upload safeguards and Messaging improvements
 date: 2025-01-31
 ---
 
-Appwrite Console now includes database metrics and better data protection features to help you monitor and manage your apps.
+Appwrite Cloud now includes new warning system for active uploads and new improvements to help you manage your apps more effectively.
```

Comment:
```suggestion
Appwrite Cloud now includes a new warning system for active uploads and improvements to help you manage your apps more effectively.
```

Reply:
@ebenezerdon this might read a little better, what do you think?

Reply:
Agreed